mandelbulber2/src/fractal_list.cpp

```diff
@@ -246,7 +246,10 @@ void DefineFractalList(QList<sFractalDescription> *fractalList)
 		Mandelbulb3Iteration, analyticDEType, logarithmicDEFunction, cpixelEnabledByDefault, 10,
 		analyticFunctionLogarithmic, coloringFunctionDefault));
 	fractalList->append(sFractalDescription("Mandelbulb 4", "mandelbulb4", mandelbulb4,
-		Mandelbulb4Iteration, analyticDEType, logarithmicDEFunction, cpixelEnabledByDefault, 10,
+		Mandelbulb4Iteration, analyticDEType, logarithmicDEFunction, cpixelEnabledByDefault, 2,
```

Comment:
line 249,  changing existing formula Mandelbulb4Iteration bailout 10 changed to bailout 2.  typo??